src/selectivity-multiple.js

```diff
@@ -107,10 +107,13 @@ var callSuper = Selectivity.inherits(MultipleSelectivity, {
         'change .selectivity-multiple-input': function() { return false; },
         'click': '_clicked',
         'click .selectivity-multiple-selected-item': '_itemClicked',
+        'mouseover .selectivity-multiple-input-container': 'mouseover',
+        'mouseleave .selectivity-multiple-input-container': 'mouseout',
```

Comment:
As far as I know we can't have events variable in the base module since we override it with the extending module. At least it's not supported by default.